pom.xml

```diff
@@ -61,7 +61,7 @@
         <version.httpcomponents-httpclient>4.5.5</version.httpcomponents-httpclient>
         <version.httpcomponents-httpcore>4.4.8</version.httpcomponents-httpcore>
         <version.in-memory-accumulo>2.0.2</version.in-memory-accumulo>
-        <version.infinispan>12.1.0.Dev01</version.infinispan>
+        <version.infinispan>10.1.8.Final</version.infinispan>
```

Comment:
Dependabot overshot the upgrade requirement to fix the issue. It's fixed in 10.0 and up, and 10 is less problematic for our existing code